edgedb-client/src/builder.rs

```diff
@@ -48,6 +48,13 @@ pub const DEFAULT_POOL_SIZE: usize = 10;
 pub const DEFAULT_HOST: &str = "localhost";
 pub const DEFAULT_PORT: u16 = 5656;
 
+#[derive(Debug, Clone, Copy, PartialEq)]
+enum TlsClientSecurity {
```

Comment:
This becames a public enum, because is visible though Credentials. So also rename it `TlsSecurity` probably (we're in the client module already), or at least to `ClientTlsSecurity` like the setting.